python/lsst/cp/pipe/linearity.py

```diff
@@ -565,6 +662,13 @@ def run(self, inputPtc, dummy, camera, inputDims,
 
             linearizer.fitResiduals[ampName] = residuals
 
+            finite = np.isfinite(residuals)
+            if finite.sum() == 0:
+                sigmad = np.nan
+            else:
+                sigmad = median_abs_deviation(residuals[finite]/inputOrdinate[finite], scale="normal")
+            linearizer.fitResidualsSigmaMad[ampName] = sigmad
```

Comment:
I'm not clear on the logic here.  Is this checking for NaN (in which case any NaN value will result in `sigmad=NaN`), or is this checking for infinite values (which I see `scipy.stats.median_abs_deviation` does handle).